### PR TITLE
Add explanation of how to escape curly brackets.

### PR DESCRIPTION
**Pull request by terrycloth**
I thought adding this to the README would be useful, after reading [issue 162](https://github.com/atom/snippets/issues/162).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -56,3 +56,18 @@ You can also use multi-line syntax using `"""` for larger templates:
       }
     """
 ```
+
+### Escaping Characters
+
+Including a literal a closing brace inside the text provided by a snippet's tab stop will close
```
Comment (50Wliu) [inline README.md]: `Including a literal closing brace`

File: README.md [hunk 2]
```diff
@@ -56,3 +56,18 @@ You can also use multi-line syntax using `"""` for larger templates:
...
+that tab stop early. You need to escape it -- with *two* backslashes, like this:
```
Comment (50Wliu) [inline README.md]: Does `To prevent that, escape the brace with two backslashes, like so:` sound better to you?

Comment (terrycloth) [inline README.md]: Sure, that works.

**Comment by 50Wliu** [comment]
Looks good to me.
/cc @atom/feedback

**Comment by izuzak** [comment]
Thanks for this, @terrycloth :cake: (Note: I converted the example into a full snippet example in 47769cdde2969faf64eb552fb15382141cb57c4f.)

**Pull request merged**